test/task.h

```diff
@@ -184,6 +184,7 @@ enum test_status {
 #  define inline __inline
 # endif
 
+# if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
Would it make sense to guard it like:

```c
#ifndef snprintf

inline int snprintf(..blah..)

#endif
```

feature detection vs. version detection. :)